src/schema/v2/me/myCollection.ts

```diff
@@ -58,6 +60,11 @@ export const MyCollection: GraphQLFieldConfig<any, ResolverContext> = {
         },
       }),
     },
+    artistIDs: {
+      type: new GraphQLList(GraphQLString),
```

Comment:
**minor:**
```typescript
      type: new GraphQLNonNull(new GraphQLList(GraphQLString)),
```

Reply:
I think what you're looking for is `new GraphQLList(new GraphQLNonNull(GraphQLString))`? Meaning: this argument isn't required but if it is you want a real array of strings not an array of strings + nulls?